joint_angles:
    init_pose: [0.022, 0.011, -0.016, 0.023, 0.120]
    init_pose_1: [2.522, 0.011, -0.016, 0.023, 1.371]
    init_pose_2: [4.523, 0.011, -0.01, 0.023, 2.942]
    init_pose_3: [4.523, 0.452, -0.83, 0.770, 2.941]
    init_pose_4: [4.523, 0.169, -1.56, 2.177, 2.941]
    rest_pose: [4.523, 0.060, -1.48, 3.355, 2.941]
    tile_0_0: [4.2135, 1.3662, -0.543, 2.7030, 2.5593]
    tile_0_1: [4.3079, 1.6757, -1.014, 2.8167, 2.6694]
    tile_0_2: [4.3547, 2.1963, -2.010, 3.3108, 2.7403]
    tile_1_0: [4.6250, 1.2850, -0.448, 2.6590, 2.9940]
    tile_1_1: [4.6021, 1.6128, -0.919, 2.7507, 2.9384]
    tile_1_2: [4.5862, 2.1064, -1.813, 3.1855, 2.9645]
    tile_2_0: [5.0206, 1.3821, -0.599, 2.7156, 3.3917]
    tile_2_1: [4.8834, 1.6961, -1.054, 2.8167, 3.3002]
    tile_2_2: [4.8180, 2.2226, -2.039, 3.2965, 3.2523]
    tile_middle_intermediate: [4.5603, 1.3959, -1.222, 3.3124, 2.9427]
    tile_left_intermediate: [4.3196, 1.3959, -1.222, 3.3123, 2.9426]
    tile_right_intermediate: [4.8474, 1.3959, -1.222, 3.3123, 2.9426]
    trigger_tile: [3.9848, 1.9422, -1.480, 3.0625, 2.3753]
    trigger_tile_intermediate: [3.9848, 1.7998, -1.555, 3.2429, 2.3753]
gripper_positions:
    open: [0.0115, 0.0115]
    close: [0.0, 0.0]
